core/src/main/java/tc/oc/pgm/rotation/vote/VotePoolOptions.java

```diff
@@ -32,32 +35,42 @@ public boolean toggleMode() {
     return replace;
   }
 
+  public boolean canAddVote() {
+    return customVoteMaps.size() < MapVotePicker.MAX_VOTE_OPTIONS;
+  }
+
   public boolean addVote(MapInfo map) {
```

Comment:
this is very confusing naming, you're not **adding a vote** you're **adding a map**